src/altinn-app-frontend/src/components/base/CheckboxesContainerComponent.tsx

```diff
@@ -181,13 +190,13 @@ export const CheckboxContainerComponent = ({
             {calculatedOptions.map((option, index) => (
               <React.Fragment key={option.value}>
                 <FormControlLabel
+                  tabIndex={-1}
```

Comment:
Bare for å sjekke.. Gjøres dette fordi hele gruppen med checkboxer/radioknapper fikk tastaturfokus? :thinking: Enig i at det var uheldig, i såfall.

Reply:
Når man klikker på flere checkbokser i samme gruppe etter hverandre, så kalles i utgangspunktet onBlur eventet for hver checkbox, dermed vil den lagre state umiddelbart flere ganger etter hverandre, som både er spam til backend, og kan føre til race-conditions med useEffecten i `useDelayedSavedState` som oppdateres når `formValue` endres. HandleBlur-funksjonen sjekker om elementet som mottar fokus er et child av checkbox-gruppa, og kjører kun `saveValue` dersom fokus flyttes utenfor gruppa som dermed løser dette problemet.

```typescript
const handleBlur = (event: React.FocusEvent<HTMLInputElement>) => {
  // Only set value instantly if moving focus outside of the checkbox group
  if (!event.currentTarget.contains(event.relatedTarget)) {
    saveValue();
  }
};
```

`event.currentTarget` er selve checkboks-gruppa, og `event.relatedTarget` er elementet som mottar fokus. Et problem som oppstår nå er at checkbox-labelen ikke kan motta fokus, og dermed blir `relatedTarget=null` når man trykker på labelen istedenfor knappen (fokus flyttes videre til knappen, men registreres ikke av eventet). Ved å sette `tabIndex={-1}` på labelen, vil ikke `relatedTarget` lenger være null når man klikker på den, og den vil ses som et element inne i gruppa og `saveValue` blir ikke kjørt. Fokus vil fortsatt flyttes til knappen, og det er ikke mulig å tabbe til label eller aktivt fokusere på den, så det burde ikke ha noen konsekvenser for tilgjengelighet.